rootfs.go

```diff
@@ -69,9 +72,39 @@ func doRootfs(device UInstaller, args runOptionsType) error {
 		return errors.New("Error while updateing image from command line: " + err.Error())
 	}
 
-	if err = device.InstallUpdate(image, imageSize); err != nil {
-		return err
+	var installed bool
+	rp := parser.RootfsParser{
+		DataFunc: func(r io.Reader, dev string, uf parser.UpdateFile) error {
```

Comment:
Looks like a copy & paste from https://github.com/mendersoftware/mender/blob/master/mender.go#L446 can we wrap into a single func? Example:
```
func makeRootfsInstaller(device UInstallCommitRebooter, dt string, installed *bool) func(r io.Reader, dev string, uf parser.Updatefile) error {
    return func(r io.Reader, dev string, uf parser.Updatefile) error {
			if installed == nil {
				panic("install flag not passed")
			}
			if dev != dt {
				return errors.Errorf("unexpected device type [%v], expected to see [%v]",
					dev, dt)
			}

			if *installed {
				return errors.Errorf("rootfs image already installed")
			}

			log.Infof("installing update %v of size %v", uf.Name, uf.Size)
			err = device.InstallUpdate(ioutil.NopCloser(r), uf.Size)
			if err != nil {
				log.Errorf("update image installation failed: %v", err)
				return err
			}

			*installed = true
			return nil
    }
}

// then
	installed := false
	rp := parser.RootfsParser{
		DataFunc: makeRootfsInstaller(device, dt, &installed),
	}
```